views/beers.hbs

```diff
@@ -0,0 +1,19 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/stylesheets/styles.css" />
+    <title>Beers</title>
+  </head>
+  <body>
+    {{#each beersFromApi}}
+        <img src="{{this.image_url}}" alt="">
+        <h1>{{this.name}}</h1>
+        <p>{{this.description}}</p>
+        <p>{{this.tagline}}</p>
+        {{> beerpartial this}}
```

Comment:
Here, you are duplicating your code. Since you are calling the partial and the information is coming from it, there's no need to leave the `image_url`, `name`, etc.